integration-test/background/atb.js

```diff
@@ -129,6 +129,30 @@ describe('install workflow', () => {
             expect(numExtiCalled).toEqual(1)
         })
     })
+
+    describe('atb storage', () => {
+        beforeEach(async () => {
+            ({ browser, bgPage, requests, teardown } = await harness.setup())
+            await backgroundWait.forSetting(bgPage, 'extiSent')
```

Comment:
The setting name makes me sad (I've noticed it before). I guess we have to live with it though :(